pilot/pkg/serviceregistry/util/label/label.go

```diff
@@ -33,7 +34,7 @@ const (
 )
 
 // AugmentLabels adds additional labels to the those provided.
-func AugmentLabels(in labels.Instance, clusterID cluster.ID, locality, k8sNode string, networkID network.ID) labels.Instance {
+func AugmentLabels(in labels.Instance, annotations labels.Instance, clusterID cluster.ID, locality, k8sNode string, networkID network.ID) labels.Instance {
```

Comment:
This looks a little hacky by putting annotions into labels

Reply:
Endpoint doesn't have idea of "Annotations" so this seems much simpler than adding a whole new field to Endpoint. Semantically it's the same,